recipe-server/client/control_new/components/common/LoadingOverlay.js

```diff
@@ -0,0 +1,29 @@
+import { Spin } from 'antd';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { areAnyRequestsInProgress } from 'control_new/state/requests/selectors';
```

Comment:
nit: This naming feels a little verbose, maybe something a bit more succinct like `hasActiveRequests` would be better?